torch/_dynamo/variables/builder.py

```diff
@@ -1184,7 +1186,7 @@ def wrap_unspecialized_primitive(self, value):
 
                 name = self.source.name()
                 if name not in self.tx.output.frame_state:
-                    # Note - this essentially means that if this name gets reused as a tensor,
+                    # Note - this esentially means that if this name gets reused as a tensor,
```

Comment:
lol

Reply:
I think we added a spellchecker to the linter - and it did it wrong lol